src/components/Book.js

```diff
@@ -9,31 +9,38 @@ export default function Book(props) {
     <li className="bookelement" id={propsHolder.book[0]}>
       <div className="leftdiv">
         <div>
-          <p>{propsHolder.book[1][0].category}</p>
-          <p>{propsHolder.book[1][0].title}</p>
-          <p>{propsHolder.book[1][0].author}</p>
+          <p className="category">{propsHolder.book[1][0].category}</p>
+          <p className="title">{propsHolder.book[1][0].title}</p>
+          <p className="author">{propsHolder.book[1][0].author}</p>
         </div>
-        <div>
-          <button type="button">comments</button>
+        <div className="buttondiv">
+          <button type="button">Comments</button>
           <button
             type="button"
             onClick={() => {
               dispatch(deleteBooks(propsHolder.book[0]));
             }}
           >
-            remove
+            Remove
           </button>
-          <button type="button">edit</button>
+          <button type="button">Edit</button>
         </div>
       </div>
       <div className="rightdiv">
-        <div>
-          completed:
+        <div className="completebar">
+          <div className="circle" />
+          <div className="completediv">
+            <p className="completenumber">0%</p>
```

Comment:
- [ OPTIONAL ] For this I recommend for you this aricle so you can add progress circle through some great react library so check this [link](https://www.npmjs.com/package/react-circular-progressbar) to add progress instead of `0%` if you would like it 💯💪🏻🔥🔥 just follow the dcumentiation and set up it in your project 😊🚀